### ISSUE TITLE
Error when loading package in nightly

### ISSUE THREAD

**Issue by scidom** [JuliaDiff/ReverseDiffSource.jl#31]
Hi @fredo-dedup, you may have seen this (in which case feel free to close the issue), ReverseDiffSource fails to load in Julia v0.5 nightly:

```
julia> using ReverseDiffSource
WARNING: Method definition newvar() in module ReverseDiffSource at /Users/theodore/.julia/v0.5/ReverseDiffSource/src/ReverseDiffSource.jl:31 overwritten at /Users/theodore/.julia/v0.5/ReverseDiffSource/src/ReverseDiffSource.jl:34.
ERROR: LoadError: LoadError: type #colon has no field env
 in include(Core.#include, UTF8String) at /Applications/Julia-0.5.0-dev-2bb94d6f99.app/Contents/Resources/julia/lib/julia/sys.dylib:-1
 in include_from_node1(Base.#include_from_node1, ASCIIString) at /Applications/Julia-0.5.0-dev-2bb94d6f99.app/Contents/Resources/julia/lib/julia/sys.dylib:-1
 in include(Core.#include, ASCIIString) at /Applications/Julia-0.5.0-dev-2bb94d6f99.app/Contents/Resources/julia/lib/julia/sys.dylib:-1
 in include_from_node1(Base.#include_from_node1, ASCIIString) at /Applications/Julia-0.5.0-dev-2bb94d6f99.app/Contents/Resources/julia/lib/julia/sys.dylib:-1
 in eval(Core.#eval, Module, Any) at /Applications/Julia-0.5.0-dev-2bb94d6f99.app/Contents/Resources/julia/lib/julia/sys.dylib:-1
 in require(Base.#require, Symbol) at /Applications/Julia-0.5.0-dev-2bb94d6f99.app/Contents/Resources/julia/lib/julia/sys.dylib:-1
 in eval(Core.#eval, Module, Any) at /Applications/Julia-0.5.0-dev-2bb94d6f99.app/Contents/Resources/julia/lib/julia/sys.dylib:-1
while loading /Users/theodore/.julia/v0.5/ReverseDiffSource/src/base_rules.jl, in expression starting on line 8
while loading /Users/theodore/.julia/v0.5/ReverseDiffSource/src/ReverseDiffSource.jl, in expression starting on line 53
```

**Comment by fredo-dedup** [comment]
Thanks for pointing this out, I hadn't seen it. 
I am currently working on a rewrite of the package, so correcting this specific issue on julia v0.5 will probably not be worth it (unless julia 0.5 gets closer to a release and I made no progress on the rewrite).

**Comment by scidom** [comment]
Of coruse, @fredo-dedup, if you are working towards such direction, it is understood there is no point to fix this issue now, feel free to close it in this case.

**Comment by fredo-dedup** [comment]
Fixed with 0.2.3  (with current latest at least)
